_field-types/supported-field-types/numeric.md

```diff
@@ -59,6 +59,38 @@ PUT testindex/_doc/1
 ```
 {% include copy-curl.html %}
 
+## Example: Skip list
+
+Use the `skip_list` parameter for better query performance. The `skip_list` parameter is particularly beneficial for fields that are frequently used in `range` queries or aggregations, because it allows the query engine to skip over document ranges that don't match the query criteria.
```

Comment:
```suggestion
Use the `skip_list` parameter for better query performance. The `skip_list` parameter is particularly beneficial for fields that are frequently used in `range` queries or aggregations because it allows the query engine to skip over document ranges that don't match the query criteria.
```